Overview.src.html

```diff
@@ -688,25 +705,25 @@ <h4 id=terminology-headers>Headers</h4>
   <tr>
    <td>"<code title>fetch</code>"
    <td>""
-   <td>"<code title>subresource</code>"
+   <td>"<code title>fetch</code>"
    <td><code title>connect-src</code>
-   <td><code title=dom-global-fetch>fetch()</code>, Cache API?
+   <td><code title=dom-global-fetch>fetch()</code>, <code>XMLHttpRequest</code>, Cache API?
```

Comment:
Presumably developers can still override Accept-Language, correct? I don't have a strong preference either way, but collapsing these does make sense to me.